tools/vf-component-library/src/site/design-principles/index.njk

```diff
@@ -0,0 +1,140 @@
+      </a>
+    </li>
+  </ul>
+</div>
+</section>
+
+<section class="embl-grid">
+  <div class="vf-section-header" id="Sustainable">
+    <h2 class="vf-section-header__heading">Sustainable</h2>
+  </div>
+  <div class="vf-content | vf-stack vf-stack--400">
+    <h3 style="margin-top: 0;">Make it Sustainable</h3>
+    <p>We will create a long-term, sustainable system. Designed for efficiency, re-use, and scale.</p>
+
+    <p>Sustainable systems are diverse and productive indefinitely. Being holistically focussed on how EMBL interacts with its audiences internal and external, means we create meaningful, sustainable experiences over the longer term.</p>
+  </div>
+</section>
+<hr class="vf-divider">
+<section class="embl-grid">
+  <div class="vf-section-header" id="Visibke">
+    <h2 class="vf-section-header__heading">Visible</h2>
+  </div>
+  <div class="vf-content">
+    <h3 style="margin-top: 0;">Show Our Work</h3>
+    <p>We will prioritise demonstration over discussion. Through prototyping we will show how these principles and values apply to the work.</p>
+
+    <p>We will embrace modern design practices by starting small and iterating quickly. We will avoid big reveals and develop a process of publishing early and often. This requires a change of mindset of the team: to be comfortable with ‘not done yet’; to be satisfied to prioritise function over form; to get the things we make in the hands of our audience so we can understand their needs more quickly.</p>
+  </div>
+</section>
+<hr class="vf-divider">
+<section class="embl-grid">
+  <div class="vf-section-header" id="Simple">
+    <h2 class="vf-section-header__heading">Simple</h2>
+  </div>
+  <div class="vf-content">
+  <h3 style="margin-top: 0;">Keep It Simple</h3>
+    <p>We will do the difficult work to keep the things we make simple, approachable, and put the user’s needs first.</p>
+
+    <p>EMBL is complex. As are the audiences, services, applications, and communications material that operate within the brand. We will do the hard work to uncover and expose the simplicity of the core.</p>
+  </div>
+</section>
+<hr class="vf-divider">
+<section class="embl-grid">
+  <div class="vf-section-header" id="Unified">
+    <h2 class="vf-section-header__heading">Unified</h2>
+  </div>
+  <div class="vf-content">
+  <h3 style="margin-top: 0;">We are all one institute</h3>
+      <p>We will cultivate cross-pollination of services, products, ideas, and collaboration.</p>
+
+      <p>We will proactively seek out opportunities to collaborate on the design of services across all channels throughout EMBL and invite feedback and new perspectives. We will share and discuss our methods.</p>
+  </div>
+</section>
+<hr class="vf-divider">
+<section class="embl-grid">
+  <div class="vf-section-header" id="Syncronised">
+    <h2 class="vf-section-header__heading">Syncronised</h2>
+  </div>
+  <div class="vf-content">
+    <h3 style="margin-top: 0;">Physical, digital, and environmental are in sync</h3>
```

Comment:
`style="margin-top: 0;"` 

_insert snarky remark_ :P

Reply:
yeah - there's a problem with `vf-content` when it starts with anything other than an `<h1>`. 

It's something we should look at in a bigger 'refactor' of `vf-content`

Reply:
I'd vote to drop the inline styles to help encourage us to fix it. But I don't feel super strongly on it.